app/screens/AppNavigator/screens/Balances/ConvertScreen.tsx

```diff
@@ -193,11 +194,14 @@ function ToggleModeButton (props: { onPress: () => void }): JSX.Element {
   )
 }
 
-function TokenVsUtxosInfo (): JSX.Element {
+function TokenVsUtxosInfo (props: { navigation: NavigationProp<BalanceParamList>}): JSX.Element {
```

Comment:
navigation prop is in the context , use `useNavigation()` instead. When `navigation` passed around, it allow unpredictable user flow.

Reply:
Thanks for the tips @ivan-zynesis! 
Could you also share some examples of the weird user flow so I can take note in the future?

Reply:
There are multiple types  of `navigation` prop. Our app now has `stack` then `bottom tab` as one of the page inside the stack. The component only inherit the access based on where itself located. eg a page inside the outer stack should not really navigate directly into a specific tab as the tab container has its on state to maintain.